wazo_dird/plugins/google_backend/services.py

```diff
@@ -185,6 +186,28 @@ def _extract_numbers_by_label(cls, contact):
 
         return numbers
 
+    @classmethod
+    def _extract_numbers_except_label(cls, contact):
+        numbers_by_label = cls._extract_numbers_by_label(contact)
+        numbers = {}
+
+        for type_ in (
+            'home',
+            'work',
+            'mobile',
+            'other',
+            'main',
+            'home_fax',
+            'work_fax',
+            'google_voice',
+            'pager',
+        ):
```

Comment:
Should be possible to use keys?
```suggestion
        for type_ in numbers_by_label.keys():
```

Reply:
No: in case the contact has only a `home` phone, the `mobile` key will not be present in the google contact, and will not be present either in `numbers_by_label`. So `numbers_except_label[mobile][0]` will return nothing (due to a `KeyError: mobile`) instead of returning the `home` phone